src/components/JsonEditorContainer/JsonEditorContainer.tsx

```diff
@@ -68,13 +69,13 @@ export function JsonEditorContainer(props: { json: any, templates: Template[] })
                 parentPath.pop();
             }
 
-            const isRemoveValid = removeNode([...path]);
+            const removePossibility = removeNode(currentJson,[...path]);
             const validInsertItems = Object(addNode([...parentPath]).resultNode?.data);
             const existingKeys: (number | string)[] = getExistingKeys(currentJson, [...parentPath]);
 
             // if removeNode validation returns error
             // Remove default Remove(Delete) function
-            if (isRemoveValid.error) {
+            if (removePossibility.error && (removePossibility.error.type !== ErrorType.InvalidPath)) {
```

Comment:
Do we have to check the error type here?
as we just have to alert the error and disable the default remove function in the next line

Reply:
Invalid error paths should not be treated as errors. It is handled here and you can handle alerts with your PR.

Reply:
Will fix with my next PR